crates/uv/src/commands/self_update.rs

```diff
@@ -74,14 +74,24 @@ pub(crate) async fn self_update(printer: Printer) -> Result<ExitStatus> {
     // available version of uv.
     match updater.run().await {
         Ok(Some(result)) => {
+            let version_information = if result.old_version.is_some() {
```

Comment:
You can use let with a pattern to avoid unwrapping, e.g., `if let Some(old_version) = result.old_version`

Reply:
I didn't know (or forgot?) that it was possible to use that while defining a variable, really nice, thanks!